### PR TITLE
OnlyOffice documentation for license path

### PR DESCRIPTION
**Pull request by voroyam**
fixes https://github.com/owncloud/docs/issues/2492

### CODE REVIEW

File: modules/admin_manual/pages/appliance/configuration/office.adoc [hunk 1]
```diff
@@ -168,6 +168,30 @@ Here are the required steps:
...
+
+When you are a Debian based Linux distributive user, the license.lic file must be placed in the following folder:
+
+----
+/var/www/onlyoffice/Data/license.lic
+----
+
+Now you will have access to the enterprise features of onlyoffice.
+
+Additional information can be found https://helpcenter.onlyoffice.com/installation/docs-enterprise-install-ubuntu.aspx[in the onlyoffice documentation]
```
Comment (mmattel) [inline modules/admin_manual/pages/appliance/configuration/office.adoc:183]:
```suggestion
== Registering your ONLYOFFICE Enterprise Edition

If you purchased the ONLYOFFICE Enterprise Edition and received the **license.lic** file, you need to import it to unlock the enterprise features.

When you are a Debian based Linux distribution user, the license.lic file must be placed into the following folder:

----
/var/www/onlyoffice/Data/license.lic
----

Doing so, you will have access to the enterprise features of ONLYOFFICE.

Additional information can be found in the https://helpcenter.onlyoffice.com/installation/docs-enterprise-install-ubuntu.aspx[ONLYOFFICE Documentation]
```

File: modules/admin_manual/pages/appliance/configuration/office.adoc [hunk 2]
```diff
@@ -168,6 +169,30 @@ Here are the required steps:
...
+
+=== Docker
+
+When you are using the docker container, the license.lic file can be placed to any folder (e.g. /app/onlyoffice/DocumentServer/data/license.lic), but you will need to mount this folder when the container starts:
```
Comment (mmattel) [inline modules/admin_manual/pages/appliance/configuration/office.adoc:188]:
```suggestion
When you are using the docker container, the `license.lic` file can be placed to any folder (e.g. /app/onlyoffice/DocumentServer/data/license.lic), but you will need to mount this folder when the container starts:
```

File: modules/admin_manual/pages/appliance/configuration/office.adoc (hunk 1 above)
Comment (mmattel) [reply modules/admin_manual/pages/appliance/configuration/office.adoc:184]: ONLYOFFICE is a proper name and needs to be written in capital letters as it is done on their homepage

Comment (voroyam) [reply modules/admin_manual/pages/appliance/configuration/office.adoc:184]: agreed

**Review by mmattel — APPROVED** [review]

**Comment by mmattel** [comment]
Backport to 10.5 and 10.6 necessary

**Comment by voroyam** [comment]
This is even better

https://helpcenter.onlyoffice.com/installation/docs-enterprise-activation.aspx

image

File: modules/admin_manual/pages/appliance/configuration/office.adoc [hunk 3]
```diff
@@ -1,6 +1,7 @@
 = Working on Documents in the ownCloud Appliance
 :toc: right
 :page-aliases: appliance/office.adoc
+:onlyoffice-es-install-url: https://helpcenter.onlyoffice.com/installation/docs-enterprise-install-ubuntu.aspx
```
Comment (mmattel) [inline modules/admin_manual/pages/appliance/configuration/office.adoc:4]:
```suggestion
:onlyoffice-es-install-url: https://helpcenter.onlyoffice.com/installation/docs-enterprise-activation.aspx
```

File: modules/admin_manual/pages/appliance/configuration/office.adoc [hunk 4]
```diff
@@ -168,6 +169,30 @@ Here are the required steps:
...
+=== Docker
+
+When you are using the docker container, the `license.lic` file can be placed to any folder (e.g. /app/onlyoffice/DocumentServer/data/license.lic), but you will need to mount this folder when the container starts:
+
+----
+docker run -itd -p 80:80 --restart=always \
+-v /app/onlyoffice/DocumentServer/data:/var/www/onlyoffice/Data \
+onlyoffice/documentserver-ie
+----
+
```
Comment (mschreml) [inline modules/admin_manual/pages/appliance/configuration/office.adoc:195]:
```suggestion
== Registering your ONLYOFFICE Enterprise Edition

If you purchased the ONLYOFFICE Enterprise Edition and received the `license.lic` file, you need to import it to unlock the enterprise features.

In Univention appliance, the `/var/lib/univention-appcenter/apps/onlyoffice-ie/Data` directory on the host computer is mounted to the `/var/www/onlyoffice/Data` data volume in the container, so the `license.lic` file can be placed to the following folder on the host computer:
```
/var/lib/univention-appcenter/apps/onlyoffice-ie/Data/license.lic
```
After the file is placed there, your version of ONLYOFFICE Enterprise Edition will become registered and you will have access to the enterprise features of ONLYOFFICE.

Additional information can be found in the {onlyoffice-es-install-url}[ONLYOFFICE Documentation]
```

**Comment by mschreml** [comment]
new PR on this: #3290

**Pull request closed without merging**